chart/validator-plugin-oci/values.yaml

```diff
@@ -51,8 +51,12 @@ metricsService:
     targetPort: https
   type: ClusterIP
 auth:
-  secretName: aws-creds
-  createSecret: true
-  accessKeyId: ""
-  secretAccessKey: ""
-  sessionToken: ""
+  secrets:
+    # Specify the names of secrets in your cluster that contain a registries credentails.
+    # E.g.: https://github.com/spectrocloud-labs/validator/blob/main/chart/validator/templates/plugin-secret-oci.yaml
+    # - secretName: oci-registry-creds
+    # - secretName: ecr-registry-creds
```

Comment:
wanted to get thoughts on the way this is setup. auth secrets for the oci plugin are a little different than the other plugins because a user could be validating against an arbitrary number of registries which could each require their own set of auth creds. with the current approach, each defined secretName will map to a separate secret resource. 

let me know if there's a better approach to handling multiple creds. other options i can think of include:
1. only support 1 secret, but that limits users to only validating against 1 private registry
2. we can store all the different creds in a single secret, but i think that'll add some complexity to our reconciler since the API would need to be updated. our API would require a secretName (which it already requires) along with some new keyname which would be used to differentiate between the creds stored in the secret.

also as an aside, we're going to need to create the plugin-secret-oci.yaml file in the validator repo. it doesn't exist at this point.

Reply:
Agreed that each registry should have its own secret. But I think creating those secrets is out of scope from the perspective of the plugin's Helm chart.

Regarding the OCI plugin secret(s) in validator, we could support an array of secret objects consisting of a secret name + credentials under the `pluginSecrets` section for the OCI plugin.

Reply:
ok, so in that case we should be able to fully remove the logic for naming the secrets from here and the users should just reference the name of the secret (created by the validator) within their `OciRegistryRule`